torch/csrc/cuda/comm.cpp

```diff
@@ -144,7 +144,7 @@ tensor_list2d broadcast_coalesced(TensorList tensors, IntArrayRef devices, size_
           // See NOTE [ Version Counter in comm.*_coalesced ]
           AT_ASSERT(t.is_variable());
           Variable var = t;
-          device_outputs.push_back(make_variable(var.data(), false));
+          device_outputs.push_back(make_variable(var, false));
```

Comment:
It's okay to remove `.data()` here, because `make_variable(tensor)` always ignores `tensor`'s original AutogradMeta and creates a new AutogradMeta.